src/opm/output/eclipse/AggregateUDQData.cpp

```diff
@@ -218,8 +226,110 @@ namespace {
         }
     } // iUap
 
+    namespace dUdw {
+
+        Opm::RestartIO::Helpers::WindowedArray<double>
+        allocate(const std::vector<int>& udqDims)
+        {
+            using WV = Opm::RestartIO::Helpers::WindowedArray<double>;
+            int nwin = std::max(udqDims[9], 1);
+            return WV {
+                WV::NumWindows{ static_cast<std::size_t>(nwin) },
+                WV::WindowSize{ static_cast<std::size_t>(udqDims[8]) }
+            };
+        }
+
+        template <class DUDWArray>
+        void staticContrib(const Opm::SummaryState& st,
+                           const std::vector<std::string>& wnames,
+                           const std::string udq,
+                           const std::size_t nwmaxz,
+                           DUDWArray&   dUdw)
+        {
+            //initialize array to the default value for the array
+            for (std::size_t ind = 0; ind < wnames.size(); ind++) {
```

Comment:
For situations like this I prefer to initialize to the default - i.e. `-0.3E+21` in this case - first and then subsequently set the elements where you have values - as:
```C++
for (std::size_t ind = 0; ind < nwmaxz; ind++)
     dudw[ind] = -0.3E+21;

for(std::size_t ind = 0; ind < wnames.size(); ind++) {
     if (st.has_well_var(wnames[ind], udq))
         dudw[ind] = st.get_well_var(wnames[ind], udq);
}
```
slightly less logic that way - no big deal.

Reply:
OK - will be changed